scripts/lisk.sh

```diff
@@ -384,66 +369,97 @@ parse_option() {
 	done
 }
 
-parse_option "$@"
-network
+# Parses network
+case $2 in
+	"mainnet")
+		NETWORK="mainnet"
+		PM2_CONFIG="$PM2_MAIN_CONFIG"
+		LISK_CONFIG="./mainnet/config.json"
+		echo -e 'Lisk configured for '"$NETWORK"' network\n' >> "$SH_LOG_FILE"
+		;;
+	"testnet")
+		NETWORK="testnet"
+		PM2_CONFIG="$PM2_TEST_CONFIG"
+		LISK_CONFIG="./testnet/config.json"
+		echo -e 'Lisk configured for '"$NETWORK"' network\n' >> "$SH_LOG_FILE"
```

Comment:
the line can be moved to after switch case section (line 393 in current codebase) as it is the same for every action